src/wp-includes/formatting.php

```diff
@@ -5082,6 +5084,9 @@ function _print_emoji_detection_script() {
 		'svgExt' => apply_filters( 'emoji_svg_ext', '.svg' ),
 	);
 
+	$version   = 'ver=' . get_bloginfo( 'version' );
```

Comment:
Where does this get used?

Reply:
It doesn't - good spot. I suspect it's a backport artefact, removed in updated PR.